packages/firebase/index.d.ts

```diff
@@ -6949,15 +6949,49 @@ declare namespace firebase.messaging {
    *
    */
   interface Messaging {
+    /**
+     * Deletes the only registration token associated with this messaging instance and unsubscribes
+     * this messaging instance from the push subscription.
+     *
+     * @return The promise resolves when the token has been successfully deleted.
+     */
+    deleteToken(): Promise<boolean>;
+
     /**
      * To forcibly stop a registration token from being used, delete it
      * by calling this method.
      *
      * @param token The token to delete.
      * @return The promise resolves when the token has been
      *   successfully deleted.
+     *
+     * @deprecated Use deleteToken() instead.
      */
     deleteToken(token: string): Promise<boolean>;
+
+    /**
+     * Subscribes the user to push notifications. Returns an FCM registration
+     * token that can be used to send push messages to the user.
```

Comment:
I would avoid using the term "user" here since the token identifies the instance and not the user in particular.

Reply:
good point. I changed it to "messaging instance" for accuracy

Reply:
I'm not currently seeing the updates Kai.  Do you need to push your latest?